### PR TITLE
test: unit test for steps & switch & load-more

### PR DESCRIPTION
**Pull request by KamenRider41**
1. I have switched the unit test implementation of the Switch and Steps components from enzyme to testing-library.
2. I have increased the unit test coverage of the load-more component.

### CODE REVIEW

File: packages/arcodesign/components/load-more/__test__/index.spec.js [hunk 1]
```diff
@@ -54,10 +54,14 @@ describe('LoadMore action', () => {
 
     it('onClick listener correctly', () => {
         const onClick = jest.fn();
-        render(<LoadMore onClick = {onClick} throttle = {200} trigger = 'click'/>)
+        const { rerender } = render(<LoadMore onClick = {onClick} throttle = {200} trigger = 'click'/>)
```
Comment (TinaPeach) [inline packages/arcodesign/components/load-more/__test__/index.spec.js:57]: 组件属性赋值中间不加空格

File: packages/arcodesign/components/switch/__test__/index.spec.js [hunk 2]
```diff
@@ -14,56 +14,49 @@ mountTest(Switch, 'Switch');
 
 describe('Switch', () => {
     it('should render correctly when set as ios/android', () => {
-        const iosComponent = mount(<Switch platform="ios" />);
-        expect(iosComponent.find('.type-ios').length).toBe(1);
-        const androidComponent = mount(<Switch platform="android" />);
-        expect(androidComponent.find('.type-android').length).toBe(1);
+        const { container: container_ios } = render(<Switch platform="ios" data-testid="switch-ios" />);
+        expect(container_ios.querySelector('.type-ios')).toBeTruthy();
```
Comment (TinaPeach) [inline packages/arcodesign/components/switch/__test__/index.spec.js:18]: 驼峰命名

File: packages/arcodesign/components/load-more/__test__/index.spec.js (hunk 1 above)
Comment (KamenRider41) [reply packages/arcodesign/components/load-more/__test__/index.spec.js]: done!

File: packages/arcodesign/components/switch/__test__/index.spec.js (hunk 2 above)
Comment (KamenRider41) [reply packages/arcodesign/components/switch/__test__/index.spec.js]: done!

**Review by TinaPeach — APPROVED** [review]

**Pull request merged**